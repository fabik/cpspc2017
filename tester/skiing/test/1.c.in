200
-837962023 966070279 -837962023 -996784055
-996138664 -819648130 965661853
-982312077 311761034 963423000
-901973437 80545400 958994690
-974117791 280368843 935850682
-868830622 535089758 930694829
-971753007 816362273 921139928
-881337551 -438388605 910022222
-885744546 -593762628 899482163
-877252811 860650260 888450544
-925411963 312666922 888051009
-920643358 -526850612 872347218
-965735404 -752739308 853470441
-966398099 -147635777 840792852
-925598240 865834951 835975782
-848320700 -471885825 833675209
-994413635 190552361 831379183
-857676137 -366654771 823912974
-895554201 322039714 816870488
-969105682 828430456 803785393
-893863730 -603755439 785935892
-934026206 706535742 772112801
-846622037 470957623 771354562
-881510838 -619613918 762698829
-875702304 -615562899 734600126
-953337521 -37276134 732402667
-911718910 872262830 728441202
-979639269 -663317683 720183308
-848473025 246835968 710006996
-959854437 -784490182 695781800
-991058890 824035375 675807246
-895862783 65970071 671117720
-926975318 145477455 666099869
-997841417 409508642 641149872
-943722837 -825663783 606092364
-873938832 -141454990 597949631
-973936770 987980937 594948887
-967986043 653508028 576538705
-926356984 205652226 568114362
-879687902 144736656 558703637
-977421530 549518822 553141740
-994910796 -654551691 551221901
-955061410 -18118427 542192283
-902034195 -562552541 539563789
-848017385 216252962 530008091
-954501986 -128479565 513822232
-891470132 386224411 502895775
-843981562 -224499065 485466983
-905298417 211320032 474549550
-863974551 419032975 455111954
-980711653 350928071 447150034
-851632384 -811269336 441841746
-964003767 394626403 436162606
-892869335 125009652 435392225
-918376814 454891373 429758351
-947186337 875253495 428550784
-944256777 757437987 406006467
-890275824 268532782 399271210
-984097536 137487898 396184221
-981273084 -75119283 392691038
-896279566 667280845 376365724
-890509019 945454992 366915389
-910652970 449490450 352290339
-851410681 905453783 338765383
-849247651 882341101 338050224
-949304903 -195621655 336576734
-975239653 -320022048 332499110
-854082834 539281793 330390725
-986957867 908014171 328862769
-929763833 332389044 327965307
-956695304 -47550799 307207152
-885821182 -543343465 282255914
-850303556 -335524887 269193669
-909530198 -454515250 252445185
-953839832 300501961 233178927
-952612777 -38171377 232691408
-886060373 -492991872 228738722
-848081587 -495554610 216017070
-908812867 679809125 214945985
-867386357 866156277 206962385
-935351922 -572224067 199349055
-921205441 462180588 191563017
-898410003 -155694223 181696232
-849563502 -53514496 178966846
-976334922 -523634537 169249938
-935429407 58115643 166018385
-950898218 -436208046 151655291
-846127666 106976255 147387494
-965008374 -34273256 127985787
-934336604 108068700 109168883
-947302001 535074890 104092261
-876983131 681227392 103739582
-907718902 -459696344 81312907
-881444672 782867314 71057313
-975467449 -257664102 46005938
-962904137 -59011445 29034501
-991018850 662082554 20872482
-970869755 423716891 -753565
-927371068 -823329544 -2730235
-985372391 718313667 -20323704
-924520961 295325138 -45711432
-985501159 -35807724 -51122032
-934869113 -126552315 -61095653
-944254437 130508585 -69556115
-959504556 -224183309 -100724918
-960918152 978884055 -103883350
-845391034 112090447 -113339115
-999374322 364648725 -121749200
-906523901 774044075 -129024273
-866985539 -521215358 -137382201
-843394383 337420725 -142768447
-917921275 -396848753 -147206863
-991562954 -439589578 -169626993
-987070438 371828337 -183736010
-840704945 -163111711 -185512205
-945073128 555482292 -186635069
-993789879 95314587 -197444538
-924665554 116595449 -199539044
-927792491 653775248 -204402914
-873466538 899045493 -215345648
-969071053 578547772 -220077842
-927133986 -115997569 -230272012
-853460020 317183285 -238752800
-905465333 -803659274 -238811009
-966798531 729403518 -255649161
-851264088 -101704367 -262520548
-928295869 -17153499 -277273906
-840873642 107232164 -280032504
-942115204 -763279844 -280863086
-928635548 46024912 -281833149
-872627220 -690602618 -284305155
-880007360 439704805 -294565561
-893438475 403493601 -316104197
-881709793 -338905766 -316365541
-876739947 640013138 -326597273
-883780903 401269614 -328975512
-993380112 219606166 -335115087
-914891476 249004122 -343966409
-881996442 -416889476 -353576271
-947122599 764778882 -361307064
-984435650 -121975181 -389884854
-844627050 -633219076 -399009567
-890228546 132815986 -408612576
-956454490 -818165916 -409792014
-970995892 -80332268 -427658009
-974347858 -540496689 -434945694
-885610881 809491169 -438808424
-904542323 -411531689 -454151304
-996144993 538985300 -489060379
-959925773 600072402 -502186703
-864492703 -201707287 -502238915
-882157439 463148013 -509778582
-950139061 925382762 -519623153
-877778625 424812221 -527495955
-951060232 749912697 -534204108
-968186561 230669608 -543090938
-997350696 151342501 -579362633
-872591082 497043052 -594077427
-880044105 -337356906 -599607581
-966293201 -833007059 -618362382
-994378875 907805327 -619254117
-946048928 342870132 -642422169
-881167323 -385617569 -654046377
-887824135 -588722697 -655145555
-896301924 -616848802 -663194868
-929619690 221178217 -682025621
-863685151 370685424 -701939500
-980609666 407204202 -708216164
-964840248 452541839 -708805233
-992498457 479739347 -711457743
-856423305 538156436 -717714238
-955991096 -467937086 -719523828
-851167074 7542546 -737726690
-901988765 -716541583 -739946066
-982646098 -246835660 -742665002
-977710308 622701771 -746380795
-958804211 260425508 -757002239
-883039782 195064354 -760138380
-998430422 637253804 -771519705
-980155331 -379138473 -773151399
-850694942 22292050 -785668601
-863005236 264601301 -786617740
-916230486 -835247740 -790935917
-890276811 -701615645 -817839151
-906719011 -553618930 -820038788
-966930409 846103824 -844375135
-988744284 -288814527 -848355880
-839458041 519939581 -883019846
-855835812 257899382 -926670780
-887056982 917054472 -926855489
-871893863 -471087193 -928605630
-860452510 92215406 -933112502
-841845822 696557627 -945776838
-854971834 500152546 -952048895
-932867000 -739103625 -965450595
-919485205 744189278 -966893703
-850289211 425614105 -967498230
-984035896 -175174333 -973721749
-861755874 748087143 -974971983
-875890593 -189009577 -994478251
-945085755 -145513776 -995864799
