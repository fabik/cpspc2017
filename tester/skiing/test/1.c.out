2
-837962023 966070279
-837962023 -996784055
